src/vs/platform/workspaces/common/workspaces.ts

```diff
@@ -265,11 +260,13 @@ function doParseStoredWorkspace(path: URI, contents: string): IStoredWorkspace {
 }
 
 export function useSlashForPath(storedFolders: IStoredWorkspaceFolder[]): boolean {
-	if (isWindows) {
-		return storedFolders.some(folder => isRawFileWorkspaceFolder(folder) && folder.path.indexOf('/') >= 0);
+	if (!isWindows) {
+		return true; // Linux/macOS always use slash
 	}
 
-	return true;
+	// Only prefer slashes if other absolute paths in the
+	// workspace configuration file use slashes as well
+	return storedFolders.some(folder => isRawFileWorkspaceFolder(folder) && isAbsolute(folder.path) && folder.path.indexOf(posix.sep) >= 0);
```

Comment:
Not sure if I understand this. Comment says to use slash only if absolute folder paths use slashes but the condition reads: use slash if there is a folder with absolute path that uses backslash

Reply:
@sandy081 `posix.sep` is `/` also on Windows, can you recheck?